index.html

```diff
@@ -0,0 +1,806 @@
+              <div class="first-article-column-section-p common-color">
+                <p>LATEST</p>
+                <p>Goldman Sachs CEO/DJ Defines Tone-deaf with latest EDM</p>
+              </div>
+              <img src="./images/24.jpg" alt="article">
+            </article>
+          </li>
+        </ul>
+        <ul class="repeat-list">
+          <li>
+            <a href="#">Satoshi Nakaboto: ‘Bitcoin crashes another 10% as stock markets plunge’ </a>
+          </li>
+          <li>
+            <a href="#">Satoshi Nakaboto: ‘Bitcoin crashes another 10% as stock markets plunge’ </a>
+          </li>
+          <li>
+            <a href="#">Satoshi Nakaboto: ‘Bitcoin crashes another 10% as stock markets plunge’ </a>
+          </li>
+          <li>
+            <a href="#">Satoshi Nakaboto: ‘Bitcoin crashes another 10% as stock markets plunge’ </a>
+          </li>
+          <li>
+            <a href="#">Satoshi Nakaboto: ‘Bitcoin crashes another 10% as stock markets plunge’ </a>
+          </li>
+        </ul>
+      </section>
+      <section class="article-section-repeat">
+        <h1 class="article-header common-color"> Distract <i class="fas fa-chevron-right"></i> </h1>
+        <div class="article-section-header-bckgimg5 width-100-ipad width-100-large-screen">
+          <div class="article-section-header-bckgimg-text color-white">
+            <h5>
+              MOST POPULAR
+            </h5>
+            <h3>
+              Satoshi Nakaboto: ‘Kazakhstan to pump $700M into crypto mining’
+            </h3>
+            <p class="font-small">
+              <span>Satoshi Nakaboto </span><span>• 13 minutes ago</span>
+            </p>
+          </div>
+        </div>
+        <ul>
+          <li class="dflex">
+            <article class="dflex-justify-space-center first-article-column-section">
+              <div class="first-article-column-section-p common-color">
+                <p>LATEST</p>
+                <p>Goldman Sachs CEO/DJ Defines Tone-deaf with latest EDM</p>
+              </div>
+              <img src="./images/24.jpg" alt="article">
+            </article>
+          </li>
+        </ul>
+        <ul class="repeat-list">
+          <li>
+            <a href="#">Satoshi Nakaboto: ‘Bitcoin crashes another 10% as stock markets plunge’ </a>
+          </li>
+          <li>
+            <a href="#">Satoshi Nakaboto: ‘Bitcoin crashes another 10% as stock markets plunge’ </a>
+          </li>
+          <li>
+            <a href="#">Satoshi Nakaboto: ‘Bitcoin crashes another 10% as stock markets plunge’ </a>
+          </li>
+          <li>
+            <a href="#">Satoshi Nakaboto: ‘Bitcoin crashes another 10% as stock markets plunge’ </a>
+          </li>
+          <li>
+            <a href="#">Satoshi Nakaboto: ‘Bitcoin crashes another 10% as stock markets plunge’ </a>
+          </li>
+        </ul>
+      </section>
+    </section>
+
+    <section class="article-section last-section">
```

Comment:
This section is encroaching on the footer.

![image](https://user-images.githubusercontent.com/57726348/92636338-d18f6880-f2f4-11ea-8ddd-2dd494feec77.png)